components/script/dom/underlyingsourcecontainer.rs

```diff
@@ -55,6 +62,108 @@ impl UnderlyingSourceType {
     }
 }
 
+#[derive(JSTraceable)]
+pub struct TeeUnderlyingSource {
+    reader: DomRoot<ReadableStreamDefaultReader>,
+    stream: Dom<ReadableStream>,
```

Comment:
is this the original stream? Would be good to rename it, and add specs links where appropriate.